src/recordlinker/linking/matchers.py

```diff
@@ -55,40 +55,7 @@ class AvailableKwarg(enum.Enum):
     defined keyword arguments. However, only a subset will be used in matching,
     based on the configuration of the algorithm.
     """
-
-    SIMILARITY_MEASURE = "similarity_measure"
-    THRESHOLD = "threshold"
-    THRESHOLDS = "thresholds"
-
-
-def _get_fuzzy_params(col: str, **kwargs) -> tuple[SIMILARITY_MEASURES, float]:
-    """
-    Helper method to quickly determine the appropriate similarity measure
-    and fuzzy matching threshold to use for fuzzy-comparing a particular
-    field between two records.
-
-    :param col: The string name of the column being used in a fuzzy
-      comparison.
-    :param kwargs: Optionally, a dictionary of keyword arguments containing
-      values for a similarity metric and appropriate fuzzy thresholds.
-    :return: A tuple containing the similarity metric to use and the
-      fuzzy comparison threshold to measure against.
-    """
-    similarity_measure: SIMILARITY_MEASURES = "JaroWinkler"
-    if "similarity_measure" in kwargs:
-        similarity_measure = kwargs["similarity_measure"]
-        # Ensure the similarity measure is valid
-        if similarity_measure not in typing.get_args(SIMILARITY_MEASURES):
-            raise ValueError(f"Invalid similarity measure: {similarity_measure}")
-
-    threshold: float = 0.7
-    if "thresholds" in kwargs:
-        if col in kwargs["thresholds"]:
-            threshold = kwargs["thresholds"][col]
-    elif "threshold" in kwargs:
-        threshold = kwargs["threshold"]
-
-    return (similarity_measure, threshold)
+    pass
```

Comment:
If the only code affiliated with the class is just `pass`, do we still need to keep this structure around? Seems like something we could probably just remove and re-type throughout the rest of the codebase.

Reply:
We don't.  I was planning on cleaning this up along with a few other things (like kwargs in initial_algorithms) in #344.

Reply:
Ah perfect, that makes total sense. Feel free to ignore here!